public/lib/loadSwaggerUI.js

```diff
@@ -16,12 +21,27 @@ $(function() {
       apiKey: '',
       dom_id: 'swagger-ui-container',
       supportHeaderParams: true,
-      supportedSubmitMethods: ['get', 'post', 'put', 'delete'],
+      supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
```

Comment:
AFAICT, the flag "supportedSubmitMethods" controls whether operation's model is editable or readonly.

https://github.com/strongloop/strong-swagger-ui/blob/b92b67923612d0f855ebe2c46ffd8ca728500fd2/src/main/javascript/view/OperationView.js#L80-L83

By default, this array is set to all HTTP verbs:

https://github.com/strongloop/strong-swagger-ui/blob/b92b67923612d0f855ebe2c46ffd8ca728500fd2/src/main/javascript/SwaggerUi.js#L26-L36

To be honest, I don't completely understand the purpose of this option. I just applied the value from swagger-ui's index.html. Perhaps we can remove this line completely and use swagger-ui's default instead?